code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -236,7 +236,7 @@
 			msg += "[t_He] look[p_s()] extremely disgusted.\n"
 
 	var/apparent_blood_volume = blood_volume
-	if(skin_tone == "albino")
+	if(skin_tone == "albino" && (dna.species.id == SPECIES_HUMAN || dna.species.id == SPECIES_FELINE))
```

Comment:
Is this really the only way we can fix this? Do alien species use skin_tone at all?

Reply:
I've changed it to check for "use_skintones" instead, since it turns out a var already existed exactly for this.